tests/probes/file/test_probes_file_multiple_file_paths.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+. $builddir/tests/test_common.sh
+
+probecheck "file" || exit 255
+which strace || exit 255
+
+function check_strace_output {
```

Comment:
So basically we expect that only `/tmp/numbers` and `/tmp/letters` should be found in strace and that oscap should not evaluate further inside these directories right?

Reply:
Your understanding is correct.